xml/admin_operating_pools.xml

```diff
@@ -64,825 +65,831 @@
+  <note>
+    <para>
+      The autoscaler will take care of the remaining optional arguments. For
+      more information, see <xref linkend="op-pgs-autoscaler"/>.
+    </para>
+  </note>
   <para>
-   This section introduces practical information to perform basic tasks with
-   pools. You can find out how to list, create, and delete pools, as well as
-   show pool statistics or manage snapshots of a pool.
+   The <command>ceph osd pool create</command> command can fail if you exceed the
+   limit of placement groups per OSD. The limit is set with the option
+   <option>mon_max_pg_per_osd</option>.
   </para>
+  <variablelist>
+   <varlistentry>
+    <term>POOL_NAME</term>
+    <listitem>
+     <para>
+      The name of the pool. It must be unique. This option is required.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>PG_NUM</term>
+    <listitem>
+     <para>
+      The total number of placement groups for the pool. This option is
+      required. Default value is 8.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>PGP_NUM</term>
+    <listitem>
+     <para>
+      The total number of placement groups for placement purposes. This should
+      be equal to the total number of placement groups, except for placement
+      group splitting scenarios. This option is required. Default value is 8.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>CRUSH_RULESET_NAME</term>
+    <listitem>
+     <para>
+      The name of the CRUSH ruleset for this pool. If the specified ruleset
+      does not exist, the creation of replicated pools will fail with -ENOENT.
+      For replicated pools it is the ruleset specified by the <varname>osd
+      pool default CRUSH replicated ruleset</varname> configuration variable.
+      This ruleset must exist. For erasure pools it is 'erasure-code' if the
+      default erasure code profile is used or
+      <replaceable>POOL_NAME</replaceable> otherwise. This ruleset will be
+      created implicitly if it does not exist already.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>erasure_code_profile=profile</term>
+    <listitem>
+     <para>
+      For erasure coded pools only. Use the erasure code profile. It must be
+      an existing profile as defined by <command>osd erasure-code-profile
+      set</command>.
+     </para>
+     <para>
+      In previous versions of &productname;, it was necessary to set the number
+      of placement groups on each pool explicitly. In &productname; &productnumber;,
+      the balancer is active by default, so the number of placement groups is
+      set and adjusted automatically. For more information on the balancer,
+      see <xref linkend="mgr-modules-balancer"/>.
+     </para>
+     <note>
+       <para>
+         If for any reason the balancer has been turned off, you can calculate
```

Comment:
If for any reason the autoscaler has been disabled (`pg_autoscale_mode` set to `off`) on a pool, you can . . .